lib/generators/has_friendship/has_friendship_generator.rb

```diff
@@ -9,7 +9,10 @@ def self.source_root
   end
 
   def self.next_migration_number(path)
-    Time.now.utc.strftime("%Y%m%d%H%M%S")
+    next_num = Time.now.utc.strftime("%Y%m%d%H%M%S").to_i
+    current_num = current_migration_number(path)
+    next_num += (current_num - next_num + 1) if current_num >= next_num
```

Comment:
isn't the first part the same as `next_num = current_num + 1`? Any reason you used `+=`?